src/lib/db/feature-toggle-client-store.ts

```diff
@@ -93,6 +96,11 @@ export default class FeatureToggleClientStore
                 'fs.feature_name',
                 'features.name',
             )
+            .leftJoin(
+                this.db('feature_tag').select('*').as('ft'),
```

Comment:
This join seems weird. I think you could just do this:
```

 .leftJoin(
        'feature_tag as ft',
        'ft.feature_name',
        'features.name',
         )
```

Reply:
Thanks, mostly artifacts left from copy/pasting from other places. Addressed in https://github.com/Unleash/unleash/pull/2396/commits/22eff75b2cdf902b20a1da8df2d94a5259330046